Index: scripts/e2e.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import sys\r\n# insert at 1, 0 is the script path (or '' in REPL)\r\nsys.path.insert(1, 'VoiceRecognition/nlu')\r\nimport Watson\r\n\r\nnlu = Watson.Watson()\r\n\r\nresponse = nlu.send_message(\"What is five time five times four\")\r\nprint(nlu.get_intents(response))\r\n\r\n
===================================================================
diff --git a/scripts/e2e.py b/scripts/e2e.py
--- a/scripts/e2e.py	
+++ b/scripts/e2e.py	
@@ -1,10 +1,9 @@
 import sys
+
 # insert at 1, 0 is the script path (or '' in REPL)
-sys.path.insert(1, 'VoiceRecognition/nlu')
-import Watson
-
-nlu = Watson.Watson()
+sys.path.insert(0, 'C:\\Users\\Ali Abdul-Hameed\\PycharmProjects\\SimpleDigitalAssistant\\VoiceRecognition\\nlu')
+from Watson import Watson
+nlu = Watson()
 
 response = nlu.send_message("What is five time five times four")
 print(nlu.get_intents(response))
-
